packages/_site/src/pages/components/link/index.tsx

```diff
@@ -102,6 +102,33 @@ export default function LinkExample() {
                                 />
                             </div>
                         </div>
+
+                        <div>
+                            <ExampleBlock
+                                title="Visited"
+                                description="The visited form of a link."
+                            />
+                            <div {...stylex.props(styles.codeWrapper)}>
+                                <PreviewBlock>
+                                    <Link href="/components/link/examples">Visited</Link>
+                                </PreviewBlock>
+
+                                <CodeBlock
+                                    language="tsx"
+                                    code={`import React from 'react';
+
+import { Link } from "@controlkit/link";
+
+export default function LinkExample() {
+	return (
+		<Link href="/components/link/examples">Default</Link>
+	);
+}
+`}
```

Comment:
This won't show a visited Link by default. When a user views this a first time its going to look like a normal link until they actually visit it. This needs to have an explicit forced style and state to display correctly consistently.